autoconfigure/storage-applicationinsights/src/main/resources/zipkin-server-applicationinsights.yml

```diff
@@ -0,0 +1,7 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=applicationinsights)
+zipkin:
+  storage:
+    applicationinsights:
+      instrumentationKey: ${AI_INSTRUMENTATION_KEY:}
+      applicationId: ${AI_APPLICATION_ID:}
+      apiKey: ${AI_API_KEY:}
```

Comment:
I  ran few thousands and didn't face much issues .. except on a day or two...For higher loads .. I am thinking of rotating API_Keys ..but would like to know about AAD.